schemas/atlas/parcellationEntityVersion.schema.tpl.json

```diff
@@ -4,56 +4,49 @@
     "studyTarget"
   ],
   "required": [
-    "name",
     "versionIdentifier"  
   ],
   "properties": {
     "hasAnnotation": {
-      "_instruction": "Add the atlas annotation which this parcellation entity defines.",
+      "_instruction": "Add the atlas annotation which this parcellation entity version defines.",
       "_embeddedTypes": [
         "https://openminds.ebrains.eu/sands/AtlasAnnotation"
       ]
     },
     "hasParent": {
-      "_instruction": "Add for this parcellation entity the defined anatomical parent structure.",
+      "_instruction": "Add for this parcellation entity version the defined anatomical parent structure.",
       "_linkedTypes": [
-        "https://openminds.ebrains.eu/sands/ParcellationEntity"
```

Comment:
needs to be both. (some parents might not exist as versions)

Reply:
adopt instruction respectively (maybe just but "version" in brakets)

Reply:
Like this:
"Add for this parcellation entity version the defined anatomical parent structure (version)."?

I think it could live without the "(version)" at the end...

Reply:
yeah fine for now